src/fragments/lib-v1/auth/js/user_attributes/update_user_attributes.mdx

```diff
@@ -0,0 +1,90 @@
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { updateUserAttributes, UpdateUserAttributesOutput } from 'aws-amplify/auth';
+
+async function handleUpdateEmailAndNameAttributes(
+	updatedEmail: string,
+	updatedName: string
+) {
+	try {
+		const attributes = await updateUserAttributes({
+			userAttributes: {
+				email: updatedEmail,
+				name: updatedName,
+			},
+		});
+		handleUpdateUserAttributesOutput(attributes);
+	} catch (error) {
+		console.log(error);
+	}
+}
+
+function handleUpdateUserAttributesOutput(
+	attributes: UpdateUserAttributesOutput
+) {
+	Object.entries(attributes).forEach(([attribute, output]) => {
+		const { nextStep } = output;
+		switch (nextStep.updateAttributeStep) {
+			case 'CONFIRM_ATTRIBUTE_WITH_CODE':
+				const codeDeliveryDetails = nextStep.codeDeliveryDetails;
+				console.log(
+					`Confirmation code was sent to ${codeDeliveryDetails?.deliveryMedium}.`
+				);
+				const confirmationCode = '112233' // code retrieved from phone or email.
+				handleConfirmUserAttribute(attribute, confirmationCode);
```

Comment:
Should we reference an example function from another code block or just leave a comment about confirming the attribute with `confirmUserAttribute`?

Reply:
I'm okay either way. I see this pattern used by Flutter as well.

Reply:
I updated it with a comment instead.